assets/app/styles/_components.less

```diff
@@ -83,6 +83,7 @@
           overflow: hidden;
           text-overflow: ellipsis;
           line-height: 1.3;
+          white-space: nowrap;
```

Comment:
Looks like this style only applies to header elements under `component` (i.e. routes). The other text still wraps.